vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/descriptor/struct/LabelDescriptor.kt

```diff
@@ -14,7 +14,7 @@ import org.vitrivr.engine.core.model.types.Value
  */
 data class LabelDescriptor(
     override val id: DescriptorId,
-    override val retrievableId: RetrievableId,
+    override val retrievableId: RetrievableId?,
```

Comment:
What is the benefit of having a nullable `RetrievableId` ? So there is a moment where a label exists without linked retrievable ?

Reply:
Here i followed the same approach as StringDescriptor, LongVectorDescriptor, etc. (git blame points to @ppanopticon ) which allows functions to return Descriptors which are later assigned to retrievables... Honestly I agree this could (and maybe should) be done differently. This would require refactoring some functions that take content as input to take retrievables as input. We need to have a clear decision on this.